# Tutorial 1: Raw data to cleaned bam files 



## Goals: 

1. Process raw fastq files to cleaned bam files for variant calling or ANGSD analysis

2. Additional processing of museum samples to correct for post-mortem sequence degradation

3. Assess differences in depth between datasets and decide wether to downsample to the same depth


## Steps

1. [Demultiplex and remove adapter](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#step-1-demultiplex-and-remove-adapter) 

2. [Clean museum data](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#step-2-clean-musuem-data)
  
  2.1 [Repair sequences with bbRepair](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#21-bbrepair)
  
  2.2 [Repair sequences with MapDamage](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#22-mapdamage)

3. [Index and prepare data for variant calling](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#step-3-index-and-prepare-for-variant-calling)

4. [Assess depth and optional downsampling to equal depths](https://github.com/alexjvr1/MuseumWGSdata_TechnicalNote/blob/main/Tutorial1_Raw2bam.md#step-4-assess-depth-and-optional-downsampling-to-equal-depths)


### Step 1: Demultiplex and remove adapter 


### Step 2: Clean musuem data


#### 2.1 bbRepair


#### 2.2 MapDamage


### Step 3: Index and prepare for variant calling


### Step 4: Assess depth and optional downsampling to equal depths 













